src/main/java/ee/tuleva/onboarding/comparisons/returns/provider/FundReturnProvider.java

```diff
@@ -35,10 +35,15 @@ public Returns getReturns(Person person, Instant startTime, Integer pillar) {
             .map(
                 key ->
                     new SimpleEntry<>(
-                        key, rateOfReturnCalculator.getRateOfReturn(accountOverview, key)))
+                        key, rateOfReturnCalculator.getReturnRateAndAmount(accountOverview, key)))
```

Comment:
Can we just call it `Return`? i.e. `getReturn(...)`

Reply:
Thanks, fixed